tests/mappers/test_fs_mapper.py

```diff
@@ -24,12 +25,13 @@
 from o2a.converter.task import Task
 from o2a.converter.relation import Relation
 from o2a.mappers import fs_mapper
+from o2a.o2a_libs.property_utils import PropertySet
 
-TEST_PARAMS = {"user.name": "pig", "nameNode": "hdfs://localhost:8020"}
+TEST_JOB_PROPERTIES: Dict[str, str] = {"user.name": "pig", "nameNode": "hdfs://localhost:8020"}
+TEST_CONFIGURATION_PROPERTIES: Dict[str, str] = {}
 
 
-# pylint: disable=invalid-name
-class prepare_mkdir_commandTest(unittest.TestCase):
+class PrepareMkdirCommandTest(unittest.TestCase):
```

Comment:
This makes it difficult to find the test code using the search.

Reply:
The naming was wrong. What you should do instead is to name the test methods appropriately rather than classes. I renamed the methods properly now and they are easily searchable.
See c8bdaebb1674b8e9f2d07437286668e8d436ef9b

Reply:
I improved it even further. I moved all the prepare_<>_command tests in single class and now it is much better conforming to Python standards (and we have no unnecessary classes created).
Please resolve this @mik-laj if you think it resolves the discussion.